src/core/flow.js

```diff
@@ -329,9 +341,23 @@ export function FlowWithPhases({
     return playOrder[playOrderPos] + '';
   };
 
+  const onPhaseBeginWrap = (state, phaseConfig) => {
+    if (phaseConfig.secondsPerPhase) {
+      phaseTimer.pause();
+      phaseTimer.reset();
+      phaseTimer.Routine = () => {
```

Comment:
I'm not sure why we need a full blown Timer class with pause and reset. Can we just use `setTimeout` here? You can use the object that it returns to erase previously running timers.

Reply:
At the moment, probably you don't need a full timer class, but in future maybe you want to store some info in timer.
If you prefer i can make it a simple timeout